types/fabric/fabric-impl.d.ts

```diff
@@ -326,6 +326,9 @@ interface IObservable<T> {
      * @param eventName Event name (eg. 'after:render')
      * @param handler Function that receives a notification when an event of the specified type occurs
      */
+    on(eventName: 'mouse:up' | 'mouse:down' | 'mouse:move' | 'mouse:up:before' | 'mouse:down:before' | 'mouse:move:before' | 'mouse:dblclick' | 'mouse:over' | 'mouse:out',
+       handler: (e: IEvent<MouseEvent>) => void): T;
```

Comment:
```suggestion
    on(
        eventName:
            | 'mouse:up'
            | 'mouse:down'
            | 'mouse:move'
            | 'mouse:up:before'
            | 'mouse:down:before'
            | 'mouse:move:before'
            | 'mouse:dblclick'
            | 'mouse:over'
            | 'mouse:out',
        handler: (e: IEvent<MouseEvent>) => void,
    ): T;
```
this is actually from DT repo prettier settings run against your PR